test/index.test.js

```diff
@@ -22,11 +23,18 @@ describe('index test', () => {
 
     beforeEach(() => {
         responseMock = {
-            toJSON: sinon.stub()
+            toJSON: sinon.stub(),
+            Items: []
+        };
+        scanChainMock = {
+            limit: sinon.stub(),
+            loadAll: sinon.stub(),
```

Comment:
It was part of the scan() chain before but I realized loadAll() actually loads all entries in the table https://github.com/screwdriver-cd/datastore-dynamodb/pull/8/files#diff-168726dbe96b3ce427e7fedce31bb0bcR170